packages/replay-next/src/suspense/EventsCache.ts

```diff
@@ -51,33 +52,40 @@ export const eventCountsCache: Cache<
 
 export const eventPointsCache = createIntervalCache<
   ExecutionPoint,
-  [client: ReplayClientInterface, eventType: EventHandlerType],
+  [client: ReplayClientInterface, eventTypes: EventHandlerType[]],
   PointDescription
 >({
   debugLabel: "EventPoints",
-  getKey: (client, eventType) => eventType,
+  getKey: (client, eventTypes) => eventTypes.join(),
```

Comment:
nit: Should probably specify a separator string.

I don't think it's likely (maybe not even possible in this specific case) but without that, you could have a key conflict (e.g. "a", "bc" and "a", "b", "c would result in the same key).

Reply:
For Firefox, this will be a single string like `["event.mouse.click"]`.  For Chromium, it should be strings like `["click,DIV","click,BUTTON"]`.  Also, `Array.join()` defaults to `","` as a separator string anyway.

Reply:
TIL. Didn't realize it defaulted to "," as a separator.